src/ui/RunEditor.tsx

```diff
@@ -616,6 +613,11 @@ export class RunEditor extends React.Component<Props, State> {
         );
     }
 
+    private variableIsValidForCategory(variable: Variable, category: Option<Category>) {
+        return (variable.category == null || variable.category === map(category, (c) => c.id)) &&
```

Comment:
Isn't this just `category?.id` here? though I believe that could yield undefined instead of null, so === may not be what we want. idk, doesn't really matter that much.

Reply:
Yeah, we should do `category?.id` here. If `variable.category` is null, it'll meet the previous conditional, so we should be ok to just use `===`.